README.md

```diff
@@ -0,0 +1,164 @@
+ti-auth.js
+==========
+
+`ti-auth.js` is a small wrapper to integrate modules into the [Travel Intelligence Portal](https://travel-intelligence.com) providing:
+- **Authentication** - unauthorized users will be redirected to the TI Portal login page and will return after successful authentication.
+- **User Context** - user information, preferences and grants are made available to the application.
+- **Utility functions** - for signout and reauthorization.
+
+
+## Installation
+via Bower:
+
+    $ bower install git@github.com:travel-intelligence/ti-auth.js --save
+
+or via NPM:
+
+    $ npm install https://github.com/travel-intelligence/ti-auth.js --save-dev
+
+Alternatively you can manually download `dist/ti-auth.js` or `dist/ti-auth.min.js` and add it to your project.
```

Comment:
This got me thinking: Maybe we should include each release’s version number in a comment, along with copyright/licensing information, in a comment of the dist files?